### DISCUSSION TITLE
How to use less memory in netlify build process?

### DISCUSSION THREAD

**Discussion by AJTJ** [react-static/react-static#1672] [Q&A] [answered]
I'm building our `react-static` + `netlify-cms` website in `netlify`, but it is using too much memory.

Is there a way to limit how much memory `react-static` uses in its build process? Do I need to limit how many threads it is using as well?

I've tried setting `--max-old-space-size=4096` in several places, but this doesn't seem to be solving the issue.

Appreciate any help on this.

Thanks,

Aaron

**Comment by zmeeeeeva** [comment, marked as answer]
I encountered similar problem, try limiting `maxThreads` and `outputFileRate` in `static.config.js`.

**Comment by AJTJ** [comment]
yes, I found this in another discussion and it worked for me!
